R/geom-abline.r

```diff
@@ -82,6 +82,16 @@ geom_abline <- function(mapping = NULL, data = NULL,
 
   # Act like an annotation
   if (!missing(slope) || !missing(intercept)) {
+
+    # Warn if supplied mapping is going to be overwritten
+    if (!missing(mapping)) {
+      warning(paste0("Using both `intercept` and/or `slope` with `mapping` may",
```

Comment:
I think the word "both" is misplaced, as it refers to the two words "intercept" and "slope". Maybe write: "Using `intercept` and/or `slope` in combination with `mapping` may"